Examples/Swift/ViewController.swift

```diff
@@ -335,7 +335,7 @@ extension ViewController: NavigationViewControllerDelegate {
     // If however you would like to pause and allow the user to provide input, set this delegate method to false.
     // This does however require you to increment the leg count on your own. See the example below in `confirmationControllerDidConfirm()`.
     func navigationViewController(_ navigationViewController: NavigationViewController, shouldIncrementLegWhenArrivingAtWaypoint waypoint: Waypoint) -> Bool {
-        return false
+        return exampleMode == .multipleWaypoints ? false : true
```

Comment:
`return exampleMode != .multipleWaypoints`

Reply:
Yeah, I thought about this, but I think for the purpose of making the example easy to read, I thought I'd explicitly say true/false.